apps/dashboard/src/routes/lms/community/ask/+page.svelte

```diff
@@ -66,8 +87,21 @@
     </div>
   </div>
 
-  <div class="mb-3 p-2">
-    <TextField bind:value={fields.title} placeholder="Title" errorMessage={errors.title} />
+  <div class="mb-3 p-2 flex gap-x-5 justify-between">
+    <TextField
+      bind:value={fields.title}
+      placeholder="Title"
+      errorMessage={errors.title}
+      className="w-[75%]"
+    />
+    <Dropdown
+      class="w-[25%]"
+      size="xl"
+      label="Select Course"
+      invalid={isInvalid}
```

Comment:
btw, thinking about it now, you don't need this `isInvalid` variable.  You should just do `!!errors.courseId`. This value already holds the error state of the field, there is no need to create another variable for error state.

I am assuming you have changed the variable in the validation to `courseId` instead of `course_id`.